### DISCUSSION TITLE
i20230521 - Library does not update

### DISCUSSION THREAD

**Discussion by 59iosl30** [rern/rAudio#1373] [Bug Reports]
Hello!

I installed release i20230521 and put 2 audio files (1 ogg and 1 wav file ) in /mnt/MPD/SD and updated the library. But the library is still empty.

Did I something wrong ?

Kind regards,
Robert

**Comment by kw-sudo** [comment]
Hi,

is there a difference if you enable '_automatic library update_' (settings-player options)? 

Oliver

**Comment by 59iosl30** [comment]
> is there a difference if you enable '_automatic library update_' (settings-player options)?

Hello Oliver!

Thanks for your hint!

It doesn't matter if I have it enabled or disabled. I always get "Update in progress ...". After a while I can choose between "Update Database or files", but that doesn't work either.

Kind regards,
Robert

**Reply by G0MJW** [reply]
Same here on a 3B+. The music is on a USB. It started building the database but then failed and now it has no data at in it at all. That might be because I the backup file I used was from last year, or it might be a permissions issue, or running out of memory perhaps.

rAudio 20230620
6.1.35-3 aarch64
Raspberry Pi 3B+ Rev 1.3
BCM2837B0 • 951 MiB
4 x Cortex-A53 @ 1.40 GHz

**Reply by G0MJW** [reply]
I tried again and it has now updated, but it took a very very long time. The USB stick is a 125 Gb Sandisk Ultrafit and I am wondering if there is some compatibility issue that is making it slow. Although I have 9 playlists, none of them are appearing. They are there, in /srv/http/data/playlists. Has something changed between versions?

Hang on - I see the issue - the owner and permissions have been changed!


-rw-r--r-- 1 http http 41516 Apr 23  2021 Playlist
-rwxrwxrwx 1 mpd  mpd     68 Jun 29 10:42 Test.m3u

Worse! The format has changed as well. These are now m3u files.

**Comment by ceejayemm** [comment]
You probably need to enable (in Setup) Player/Options/FFmpeg and then rebuild your database.

Chris

**Comment by 59iosl30** [comment]
> You probably need to enable (in Setup) Player/Options/FFmpeg and then rebuild your database.

Same message "Update in progress ...".

Kind regards,
Robert

**Reply by ceejayemm** [reply]
Permissions of music files ? From readMe:

Minimum permission for music files (on Linux ext filesystem)
Directories: rwxr-xr-x (755)
Files: rw-r--r-- (644)

Chris

**Comment by 59iosl30** [comment]
> Permissions of music files ? From readMe:

SD-Folder has 0755 and two files inside of SD-Folder have 0644.

Kind regards,
Robert
